demo/src/main/java/com/google/googlemediaframeworkdemo/demo/adplayer/ImaPlayer.java

```diff
@@ -55,6 +55,8 @@
  */
 public class ImaPlayer {
 
+  private static String PLAYER_TYPE = "GMF ImaPlayer";
```

Comment:
Can we change this to something that indicates that this is Android and not iOS? Maybe "gmf_android" or something like that.